Makefile

```diff
@@ -25,12 +23,9 @@ RUST_MUSL_BIN = ${CARGO_TARGET_DIR}/x86_64-unknown-linux-musl/release
 # Location to place intermediate files and output artifacts
 # during the build process. Note the go tool ignores directories
 # with leading '.' or '_'.
-WORKDIR  = .build
+WORKDIR  = $(realpath .)/.build
```

Comment:
Can you shed some light on why this is an absolute path? It seems like it just makes it harder to work with, since the task names become dependent on the location that the repo is checked out to. I had recently removed this and made them relative. Did that break something?

Reply:
> since the task names become dependent on the location that the repo is checked out to

I'm not quite following this. What are task names (make targets), and what is the relevance of their being dependent on the repo checkout path?

The specific reason for doing this is that `go test` compiles test binaries into arbitrary temporary directories and runs them from there. These binaries won't have co-located exec's in the same directory, and require that they be available on the $PATH.

Meanwhile this PR _removes_ any dependency on `GOPATH` , which I still _think_ is desirable given the constant friction of "which binary is running? the one in ~/go/bin or the one in .build/package/bin?". But that means we can't assume that binaries will be externally configured to be available on the $PATH, and thus Makefile must arrange for $PATH must include the binaries.

But... those test binaries are running in an arbitrary temp directory as their cwd, and a relative `.build/package/bin` is meaningless for them.

Reply:
Ah, gotcha. Sounds like it's mostly just about ensuring we have the PATH set to include those. IMO it would be preferable to add the absolute path prefix when setting the PATH instead of changing PKGDIR, since PKGDIR is used in target names. I sometimes use target names on the CLI, either with `-o`, or just to build a single binary, and it's more convenient to use the relative path. Should be easy to have our cake and eat it too, though, so can just update it later.